main.cpp

```diff
@@ -243,238 +242,229 @@ struct Resturant
         float countTip(float tip);
     };
 
-    //1) serve food
     void serveFood();
-    //2) collect payment
-    float collectPayement(float payment);
-    //3) provide eating utensils
+    float collectPayment(float payment);
     void utensils();
 };
-/*
-Thing 4) car dealership
-5 properties:
-    1) number of cars in inventory(int)
-    2) amount of cars sold in a month(int)
-    3) amount sales people on staff(int)
-    4) number of cars serviced each week(int)
-    5) amount of inventory bought this month(int)
-3 things it can do:
-    1) sell cars
-    2) hire salesmen
-    3) buy inventory
- */
+
+void Resturant::serveFood()
+{
+
+}
+
+float Resturant::collectPayment(float payment)
+{
+    return payment;
+}
+
+void Resturant::utensils()
+{
+
+}
+
+void Resturant::Server::serveCust()
+{
+
+}
+
+void Resturant::Server::cleanTable()
+{
+
+}
+
+float Resturant::Server::countTip(float tip)
+{
+    return tip;
+}
+
 struct CarDealership
 {
-    //1) number of cars in inventory(int)
     int carInventory = 200;
-    //2) amount of cars sold in a month(int)
     int carSold = 50;
-    //3) amount sales people on staff(int)
     int staff = 23;
-    //4) number of cars serviced each week(int)
     int carServiced = 40;
-    //5) amount of inventory bought this month(int)
     int invBought = 15;
-    //1) sell cars
-    float CarSold(float price);
-    //2) hire salesmen
+    float carSale(float price);
     void salesmen();
-    //3) buy inventory
-    int inventor(int invBought);
+    int invator(int invatory);
 };
```

Comment:
I didn't catch these improper function names in part 1. 
What you have written here does not match your plain-english actions.
You need to include the action words in your function names
```
float sellCars(float price);
void hireSalesmen();
int buyInventory(int inventoryBought);
```
- [ ] revise